ssl/ssl_lib.c

```diff
@@ -431,6 +431,35 @@ static int dane_tlsa_add(SSL_DANE *dane,
     return 1;
 }
 
+/*
+ * Return 0 if there is only one version configured and it was disabled
+ * at configure time.  Return 1 otherwise.
+ */
+static int ssl_check_allowed_versions(int min_version, int max_version)
+{
+    if (min_version == max_version
```

Comment:
It's a tradeoff between comprehensiveness of check and complexity of code.  Handling the case where one or the other is zero gets kind of awkward.  (Kurt had asked a similar question.)  But let me know if you want me to put something together.